enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -219,6 +219,28 @@ Examples:
 
 ### User Stories [optional]
 
+* As a user unfamiliar with proprietary templating languages, I want a migration
```

Comment:
Is this related to discovery? I thought this was more on the `transformation` part.

Reply:
done